NAMESPACE

```diff
@@ -31,6 +31,7 @@ export(Phase)
 export(R_Date)
 export(R_Simulate)
 export(Sequence)
+export(as.CalDates)
```

Comment:
Sneaky! That should be in another PR :smile:

Reply:
Right, is part of another branch and sneaked its way into this one. Removed from this branch